simplq/src/components/pages/Admin/QueueSettings.jsx

```diff
@@ -64,7 +63,7 @@ export default ({ queueId }) => {
   return (
     <>
       <SidePanelItem Icon={SettingsIcon} title="Queue Settings" onClick={toggleModal} />
-      <Modal open={isModalOpen}>
+      <Dialog PaperProps={{ style: { width: '27%', borderRadius: 30 } }} open={isModalOpen}>
```

Comment:
Let's make this change [here](https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/common/Modal/Modal.jsx).

Reply:
I think now we don't need the Modal component, we can replace all the Modal with Dialog components and it will do the all work that the Modal component is doing and more

Reply:
I have now made the changes in the Modal component, Please have a look, Thank you